### DISCUSSION TITLE
Problem with workflow using ollama

### DISCUSSION THREAD

**Discussion by filosofo-justino** [langgenius/dify#7218] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I have a working dify workflow application.
In which I am using the parameter extractor block, I have groq and openrouter ai as providers using the llama3.1:8b model.
I am trying to migrate the model to use ollama on a local gpu based on pytorch/pytorch:latest docker image (I also tested with ollama on colab pro).
In both I get all empty extraction parameters followed by this error " "__is_success": 0,
  "__reason": "Invalid number of parameters","

What can I do to make the model integrate correctly to this flow?

**Comment by patryk20120** [comment]
Does the LLM block with local ollama return an answer?

**Reply by filosofo-justino** [reply]
No, the llm block does not return an answer, it returns null, but in chatbot it works.

**Comment by filosofo-justino** [comment]
Thanks Patryk for your answer, it is already solved, you have to use the models in chat mode for the extract node.

**Reply by rspilk** [reply]
I am also in chat mode and getting an output of the LLM block containing text: ""

What else did you do?
